images/access-setup/content/bin/setup_compute.sh

```diff
@@ -59,13 +67,16 @@ Optional arguments:
         Display this message.
 Example:
     %s -d -k /path/to/compute.kubeconfig
-" "${0##*/}" "$KUSTOMIZATION" "$GIT_URL" "$GIT_REF" "${0##*/}" >&2
+" "${0##*/}" "$KUSTOMIZATION" "$GIT_URL" "$GIT_REF" "$TEKTON_RESULTS_DATABASE_USER" "$TEKTON_RESULTS_DATABASE_PASSWORD" "${0##*/}" >&2
 }
 
 parse_args() {
   KUSTOMIZATION=${KUSTOMIZATION:-github.com/openshift-pipelines/pipeline-service/gitops/compute/pac-manager?ref=main}
   GIT_URL=${GIT_URL:-"https://github.com/openshift-pipelines/pipeline-service.git"}
   GIT_REF=${GIT_REF:="main"}
+  TEKTON_RESULTS_DATABASE_USER=${TEKTON_RESULTS_DATABASE_USER:="tekton"}
```

Comment:
I'm not sure those params should have default values.

Reply:
defaults removed